script/Arcueid/Day 5/05_00_ARC05_10.txt

```diff
@@ -122,7 +122,7 @@ But as someone who managed to make it out alive, I cannot allow myself to voice
 [sha:27b3d5cb3b5ce8644fdf3af1def0b8ceef15a3cc]{
 -- Page 723, Offset 10616.
 -- 新参とは言え、志貴は祖の<一角|ひとり>を殺したのよ？
-Even if he was a newcomer, you killed a member of the Ancestors, you know?
+Even if he was a young blood, you killed a member of the Ancestors, you know?
```

Comment:
revert

Reply:
I think young blood here is appropriate, it's what we decided to call the "younger" vampires at some point when me and Ross talked about it.
In fact, I think he's the one who added it to the spreadsheet. Could try and make it more clear though:
"It might've been a young blood that you killed, but it was still one of the Ancestors, you know?"